frontend/scripts/verify-e04-05.sh

```
@@ -0,0 +1,75 @@
+#!/bin/bash
+
+# Note: set -eを使用せず、明示的なエラーハンドリングを実装
+# すべてのエラーを収集してレポートするため
+
+# プロジェクトルートに移動
+cd "$(dirname "$0")/.."
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)